migrations/2_deploy_contracts.js

```diff
@@ -1,5 +1,5 @@
 module.exports = function(deployer) {
   deployer.deploy(ConvertLib);
```

Comment:
It's an example contract  that is added when calling "truffle init". It's located under contracts and I don't think we need it.